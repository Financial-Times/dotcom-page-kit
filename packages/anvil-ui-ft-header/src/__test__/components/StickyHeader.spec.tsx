import React from 'react'
import renderer from 'react-test-renderer'
import fixture from '../../__stories__/story-data/index'
import { StickyHeader as Subject } from '../../index'

const propsAnonymous = { ...fixture, userIsAnonymous: true, userIsLoggedIn: false }
const propsLoggedIn = { ...fixture, userIsAnonymous: false, userIsLoggedIn: true }

describe('anvil-ui-ft-header/src/components/StickyHeader', () => {
  it('renders as an anonymous user', () => {
    const tree = renderer.create(<Subject {...propsAnonymous} />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders as a logged in user', () => {
    const tree = renderer.create(<Subject {...propsLoggedIn} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
