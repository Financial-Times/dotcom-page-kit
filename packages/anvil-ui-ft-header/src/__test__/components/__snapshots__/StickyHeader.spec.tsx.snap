// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`anvil-ui-ft-header/src/components/StickyHeader renders as a logged in user 1`] = `null`;

exports[`anvil-ui-ft-header/src/components/StickyHeader renders as an anonymous user 1`] = `null`;
